src/main/java/org/mitre/inferno/Validator.java

```diff
@@ -64,7 +77,7 @@ public Validator(String igDir) throws Exception {
     // The two lines below turn off URL resolution checking in the validator. 
     // This eliminates the need to silence these errors elsewhere in Inferno
     // And also keeps contained resources from failing validation based solely on URL errors
-    ValidationControl vc = new BaseValidator(null, null)
+    ValidationControl vc = new BaseValidator(null, null, false)
```

Comment:
This new parameter here is a debug flag https://github.com/hapifhir/org.hl7.fhir.core/blob/eef3ebc5f1e170f67f89289fcc6cb4ca0ff9e94c/org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/BaseValidator.java#L163